syncer/online-ddl-tools/online_ddl.go

```diff
@@ -74,9 +70,11 @@ type TableType string
 
 // below variables will be explained later.
 const (
-	RealTable  TableType = "real table"
-	GhostTable TableType = "ghost table"
-	TrashTable TableType = "trash table" // means we should ignore these tables
+	RealTable       TableType = "real table"
+	GhostGhostTable TableType = "ghost ghost table"
```

Comment:
seems we could combine GhostXXX and PTXXX

Reply:
We need to distinguish them here. https://github.com/pingcap/dm/pull/1904/files/eb17460e69c7d7428d1067e01c64d71a058871a9#diff-f95255b4403338eb4e9ea2f2a8b9e6d073a6ccd4adee5d11e1410c008b1ee89bR519-R526

Reply:
`table = table[1 : len(table)-4]` should cover both types ⬆️